### DISCUSSION TITLE
Veryl support on RgGen

### DISCUSSION THREAD

**Discussion by taichi-ishitani** [veryl-lang/veryl#1110] [General]
Hi,

I'm developing a CSR automation tool named RgGen.
https://github.com/rggen/rggen

I released RgGen v0.33.4.
https://github.com/rggen/rggen/releases/tag/v0.33.4

Veryl RTL generation is officially supported from this release.
You can generate CSR modules written in Veryl from readable register map specifications.

Followings are example generated Veryl files.
* https://github.com/rggen/rggen-sample/blob/master/block_0.veryl
* https://github.com/rggen/rggen-sample/blob/master/block_1.veryl
* https://github.com/rggen/rggen-sample/blob/master/uart_csr.veryl

**Comment by dalance** [comment]
How about adding "Related Projects" section and RgGen entry to README.md?

**Reply by taichi-ishitani** [reply]
Sure. I will open a PR for this soon.
